vnext/ReactUWP/Modules/NativeUIManager.h

```diff
@@ -101,6 +101,7 @@ class NativeUIManager : public facebook::react::INativeUIManager {
   facebook::react::INativeUIManagerHost *m_host = nullptr;
   Mso::CntPtr<Mso::React::IReactContext> m_context;
   bool m_inBatch = false;
+  bool m_useLegacyStretchBehaviour = true;
```

Comment:
>true [](start = 37, length = 4)

This seems to me that this is in conflict with the description of the pr. My interpretation was that the default is new behavior and the legacy behavior is opt-in. Therefore I would have expected to see `false` here.

Reply:
-or- are the other platforms explicitly setting the `LegacyStrechth` for Yoga, and that the term legacy is from Yoga and not from React-Native-Windows?

If so, maybe a comment for folks not familiar with yoga :D

---
In reply to: [436292693](https://github.com/microsoft/react-native-windows/pull/5140#discussion_r436292693) [](ancestors = 436292693)

Reply:
Note: Same for the default of `true` on line 156 of the `.cpp` file.

---
In reply to: [436292840](https://github.com/microsoft/react-native-windows/pull/5140#discussion_r436292840) [](ancestors = 436292840,436292693)

Reply:
It looks like legacyStretchBehavior is a yoga config, not from react-native-windows. 

It appears to be set here for the android code: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactYogaConfigProvider.java#L21

Reply:
Yeah, the naming of the yoga property is confusing when looking at it from the react-native-windows point of view.
I reworked the PR some to actually expose the setting to the public interface.  Now if you want the old behavior you actually do something like:

```csharp
Microsoft::ReactNative::SetMatchAndroidAndIOSStretchBehavior(Host->Settings, false);
```

Reply:
Thanks! A lot more straight-forward!